gshoppingflux/gshoppingflux.php

```diff
@@ -1634,7 +1634,7 @@ public function renderInfo()
 		));
 	}
 
-	public function customGetNestedCategories($shop_id, $root_category = null, $id_lang = false, $active = true, $groups = null, $use_shop_restriction = true, $sql_filter = '', $sql_sort = '', $sql_limit = '')
```

Comment:
Actually I removed this parameter because it's throwing an exception because SQL Query is malformed (where clause):

```
SELECT c.*, cl.`name` as gcat_name, g.*, gl.*, s.name as shop_name
FROM `ps_category` c
INNER JOIN `ps_category_shop` cs ON (cs.`id_category` = c.`id_category` AND cs.`id_shop` = "1")
LEFT JOIN `ps_category_lang` cl ON c.`id_category` = cl.`id_category` AND cl.`id_shop` = "1"
LEFT JOIN `ps_gshoppingflux` g ON g.`id_gcategory` = c.`id_category` AND g.`id_shop` = "1"
LEFT JOIN `ps_gshoppingflux_lang` gl ON gl.`id_gcategory` = c.`id_category` AND gl.`id_shop` = "1"
LEFT JOIN ps_shop s ON s.`id_shop` = "1"
WHERE 1 ; AND cl.`id_lang` = 1 AND gl.`id_lang` = 1 AND c.`active` = 1 ORDER BY c.`level_depth` ASC, cs.`position` ASC;
```

it's adding a semicolon incorrectly.